src/client/initClient.jsx

```diff
@@ -29,6 +29,8 @@ export default async function initClient() {
   try {
     // eslint-disable-next-line no-underscore-dangle
     setModuleMap(global.__CLIENT_HOLOCRON_MODULE_MAP__);
+    // eslint-disable-next-line no-underscore-dangle
+    setRequiredExternalsRegistry(global.__HOLOCRON_EXTERNALS__);
```

Comment:
NIT: for consistency with `__CLIENT_HOLOCRON_MODULE_MAP__`, why not `__CLIENT_HOLOCRON_EXTERNALS__`?

Reply:
`__CLIENT_HOLOCRON_MODULE_MAP__` is the only key with the `CLIENT` prefix, this is probably as the module map being set is specifically for the client and differs from the module map loaded on the server.